modules/data-loading/pages/data-streaming-connector.adoc

```diff
@@ -400,12 +332,293 @@ GSQL > RUN LOADING JOB load_person
 By default, loading jobs that use Kafka data sources run in streaming mode and do not stop until manually aborted.
 As data is streamed from the data source, the running loading job will continuously ingest the streamed data into the graph store.
 
-==== Stream data source updates
+=== Stream data source updates
 
 To load updates to the data source after you've created the connectors, you need to delete the connector and recreate another connector.
 
 In the future, the streaming data connector will support automatically scanning for updates and stream data to TigerGraph.
 
+== Stream data from an external Kafka cluster
+You can create a data connector between an external Kafka cluster (source cluster) and TigerGraph’s internal Kafka server (destination cluster) to stream data from the source cluster.
```

Comment:
Please rephrase this to something like: You can create a data connector stream data from an external Kafka cluster (source cluster) to TigerGraph’s internal Kafka server (destination cluster).
Also the destination does not have to be Tigergraph's internal Kafka server. I can be any kafka cluster